runtime-sdk/Cargo.toml

```diff
@@ -25,6 +25,8 @@ impl-trait-for-tuples = "0.2.1"
 base64 = "0.13.0"
 once_cell = "1.8.0"
 slog = "2.7.0"
+evm = { version = "0.28.0", features = ["with-serde"] }
```

Comment:
Given these dependencies, maybe we should move this module into its own crate (still in the same repo, just maybe under `runtime-sdk/modules/evm`) to avoid bloating the core SDK? I'll probably do the same with the `contracts` module.